PerimeterXModule/PxModule.cs

```diff
@@ -398,6 +400,19 @@ private bool IsFilteredRequest(HttpContext context)
 				return true;
 			}
 
+			// block specific routes prefix
+			if (blockSpecificRoutes != null)
+			{
+				foreach (var prefix in blockSpecificRoutes)
+				{
+					if (url.StartsWith(prefix))
+					{
+						return false;
+					}
+				}
+				return true;
```

Comment:
Please add a comment here, as it is not entirely trivial that in fact what you are doing here is counter intuitive - deciding to filter-out routes that do _not_ have the prefix.

Reply:
👍